src/Components/Server/src/Circuits/CircuitHost.cs

```diff
@@ -133,7 +134,71 @@ internal void SendPendingBatches()
             // Dispatch any buffered renders we accumulated during a disconnect.
             // Note that while the rendering is async, we cannot await it here. The Task returned by ProcessBufferedRenderBatches relies on
             // OnRenderCompleted to be invoked to complete, and SignalR does not allow concurrent hub method invocations.
-            var _ = Renderer.Dispatcher.InvokeAsync(() => Renderer.ProcessBufferedRenderBatches());
+            _ = Renderer.Dispatcher.InvokeAsync(() => Renderer.ProcessBufferedRenderBatches());
+        }
+
+        internal async Task EndInvokeJSFromDotNet(long asyncCall, bool succeded, string arguments)
```

Comment:
`CircuitHost` is already `internal` so it's clearer if this method and the one below are `public`

Reply:
Yeah, `ctrl+.` defaults to internal. I don't feel strongly about it, so I'll change it.